src/axom/primal/tests/primal_boundingbox.cpp

```diff
@@ -40,7 +40,10 @@ void check_bb_policy()
       box[i].isValid();
     });
 
-  EXPECT_EQ(box[0], BoundingBoxType(PointType(0.0), PointType(10.0)));
+  BoundingBoxType box_host;
```

Comment:
Minor: Would this code be simpler if  `box` used a `StackArray`  instead of allocating and deallocating one bounding box?